libcodechecker/server/client_db_access_handler.py

```diff
@@ -415,6 +415,21 @@ def __sortResultsQuery(self, query, sort_types=None):
 
         return query
 
+    def __get_run_ids_to_query(self, session, cmp_data=None):
```

Comment:
Use this function at here too:
https://github.com/Ericsson/codechecker/blob/10bd1676033784630e6ebf4a848b4e19c6820539/libcodechecker/server/client_db_access_handler.py#L689-L690